tdrs-frontend/.env

```diff
@@ -0,0 +1,37 @@
+###
+# Default environment variables used by React
```

Comment:
NOTE: this file gets added with https://github.com/raft-tech/TANF-app/pull/660 but was also needed here to support not having to re-define all the variables that don't change between environments